bahmni_api_feed/models/reference_data_service.py

```diff
@@ -28,7 +28,6 @@ def _fill_data(self, vals, category=None):
         specified_product_category = vals.get("product_category")
         if specified_product_category:
             category_name = specified_product_category
-        _logger.info("\n******* Creating Product with Category: %s, hierarchy: %s", category_name, category_hierarchy)
```

Comment:
Would prefer to keep such log statements for some time.. for team to debug issues.

Reply:
sure, we will